types/office-js-preview/index.d.ts

```diff
@@ -117993,254 +117993,248 @@ declare namespace OneNote {
 
 declare namespace Visio {
     /**
-     *
      * Provides information about the shape that raised the ShapeMouseEnter event.
      *
+     * @remarks
      * [Api set:  1.1]
```

Comment:
Yeah, it comes from a different source file. I think that because the Visio API are only used to embedded diagrams in SP, isSetSupported doesn't matter (since SP Online will always be up to date).